notifications_utils/letter_timings.py

```diff
@@ -26,31 +36,26 @@ def get_letter_timings(upload_time, postage='second'):
     # shift anything after 5:30pm to the next day
     processing_day = utc_string_to_aware_gmt_datetime(upload_time) + timedelta(hours=6, minutes=30)
 
-    def next_monday(date):
-        """
-        If called with a monday, will still return the next monday
-        """
-        return date + timedelta(days=7 - date.weekday())
+    def get_next_work_day(date, bank_holidays):
```

Comment:
I think the argument `non_working_days` would be more accurate than `bank_holidays` here? As we are actually including bank holidays and 'weekends' in the days to be ignored, which isn't very clear just by the phrase `bank_holidays`